config/main.py

```diff
@@ -1554,7 +1577,13 @@ def checkpoint(ctx, checkpoint_name, verbose):
 
        <checkpoint-name>: The checkpoint name, use `config list-checkpoints` command to see available checkpoints."""
     try:
-        GenericUpdater().checkpoint(checkpoint_name, verbose)
```

Comment:
>checkpoint [](http://example.com/codeflow?start=25&length=10)

Could you explore lower level reuse? ConfigWrapper.get_config_db_as_json should be changed to handle both single-asic and multi-asic.